cmd/corectl/main.go

```diff
@@ -65,6 +65,9 @@ var commands = map[string]*command{
 	"init":                 {initCluster},
 	"evict":                {evictNode},
 	"allow-address":        {allowRaftMember},
+	"get":                  {get},
```

Comment:
@jeffomatic was suggesting using making even the read command different depending on whether the option is a tuple or a set. So like set/add vs get/*something*. Maybe *something* could be "list".

But seeing the implementation now, I'm not sure. A single tuple-value could just produce a single line of output. That seems fine to me.